### PR TITLE
Regenerates docs with roxygen2 7.1.1

### LINKED ISSUE #191: Upgrade to roxygen2 7.1.1
We are currently using roxygen2 6.1.1. Upgrade to the latest released roxygen2 (7.1.1).

The changelog for roxygen2 is at https://roxygen2.r-lib.org/news/index.html.
One interesting new roxygen2 feature is inline markdown code and code chunks using the knitr notation.

### PR DESCRIPTION
**Pull request by johnbradley**
Fixes #191

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(as.irilist,default)
```
Comment (hlapp) [inline NAMESPACE:3]: Do you know where that's coming from? I don't recall that class but my memory has lots of holes ...

**Review by hlapp — APPROVED** [review]
LGTM. Remind me where the `irilist` S3 class is coming from?

File: NAMESPACE (hunk 1 above)
Comment (johnbradley) [reply NAMESPACE:3]: Yes I do. This is my mistake. Good catch. I need to regenerate this again.

**Review by hlapp — APPROVED** [review]
👍🏻

**Pull request merged**